admin/src/components/tag-list.jsx

```diff
@@ -0,0 +1,163 @@
+import { React, useState, useEffect } from "react";
+import { Button, Row, Container, Col, Form } from "react-bootstrap";
+import List from "./util/list/list";
+import DeleteModal from "./delete-modal/delete-modal";
+
+/**
+ * The tag list component.
+ *
+ * @returns {object}
+ *   The TagList
+ */
+function TagList() {
+  const [selectedCells, setSelectedCells] = useState([]);
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const [tags, setTags] = useState([]);
+
+  /**
+   * Load content from fixture.
+   * TODO load real content.
+   */
+  useEffect(() => {
+    fetch("./fixtures/tags/tags.json")
+      .then((response) => response.json())
+      .then((jsonData) => {
+        setTags(jsonData);
+      });
+  }, []);
+
+  /**
+   * @param {object} props
+   * The props.
+   * @param {string} props.name
+   * The name of the tag.
+   * @param {number} props.id
+   * The id of the tag
+   */
+  function handleSelected({ name, id }) {
+    const localSelectedCells = [...selectedCells];
+    const alreadySelected = selectedCells.find((x) => x.id === id);
+    if (alreadySelected) {
+      localSelectedCells.splice(localSelectedCells.indexOf({ name, id }), 1);
+    } else {
+      localSelectedCells.push({ name, id });
+    }
+
+    setSelectedCells(localSelectedCells);
+  }
+
+  /**
+   * @param {object} props
+   * The props.
+   * @param {string} props.name
+   * The name of the tag.
+   * @param {number} props.id
+   * The id of the tag
+   */
+  function openDeleteModal({ id, name }) {
+    setSelectedCells([{ id, name }]);
+    setShowDeleteModal(true);
+  }
+
+  const columns = [
+    {
+      key: "pick",
+      label: "Valg",
```

Comment:
Danish mixed into this? Please use english strings :-)

Reply:
will be fixed